log/file_handler.ts

```diff
@@ -100,11 +114,4 @@ export class FileHandler extends BaseHandler {
   #resetBuffer() {
     this._pointer = 0;
   }
-
-  override destroy() {
```

Comment:
This method should have a deprecation notice. Otherwise, this is a breaking change. We can also just call `.close()` within it now.